src/Makefile

```diff
@@ -58,13 +60,15 @@ CRYPTOPP_INC_Darwin:=-I/opt/local/include
 endif
 endif
 
+PROTOC_Darwin:=/opt/local/bin/protoc
```

Comment:
No need to hardcode the PATH if we can be relative to `grpc_cpp_plugin`.
I would go with something like this.

```
GRPC_CPP_PLUGIN_Darwin:=$(shell which grpc_cpp_plugin)
GRPC_INC_Darwin:=-I$(dir $(GRPC_CPP_PLUGIN_Darwin))../include
GRPC_LIB_Darwin:=-L$(dir $(GRPC_CPP_PLUGIN_Darwin))../lib -lgrpc++ -lgrpc -lgpr -lprotobuf -lpthread -labsl_synchronization
PROTOBUF_LIB_Darwin:=-L$(dir $(GRPC_CPP_PLUGIN_Darwin))../lib -lprotobuf -lpthread
```

Reply:
sent you a patch